test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -522,6 +522,33 @@ public async Task SyncTriggers_InternalAuth_Succeeds()
             }
         }
 
+        [Theory]
+        [InlineData("", HttpStatusCode.Unauthorized)]
+        [InlineData("|", HttpStatusCode.Unauthorized)]
```

Comment:
"|" is the config value we'd use to disable for all (i.e. empty whitelist). I don't think an empty string makes it through the entire config pipeline.